ConwaysGameOfLife.Tests/ValidatorShould.cs

```diff
@@ -10,47 +9,55 @@ public ValidatorShould()
         {
             _validator = new Validator();
         }
+
         [Fact]
         public void AcceptUniverse_WhenInputIsValid()

```

Comment:
Is this name clearer? (It might not be - ignore if it isn't) If so, could adopt similar names for other tests in `ValidatorShould`
```suggestion
        public void ReturnTrue_GivenValidUniverseInput()
```